.github/workflows/run-e2e-daily.yaml

```diff
@@ -0,0 +1,32 @@
+name: Bacom E2E Daily Run
+
+on:
+  schedule:
+    - cron: '0 16 * * *'
```

Comment:
Do we want to add a `workflow_dispatch` manual run?

Reply:
Thanks for the suggestion. I added the manual run option.